templates/error.mustache

```diff
@@ -0,0 +1,6 @@
+{{!
```

Comment:
Should this file be here?

Reply:
No I intended to have this populated when I planned the pullrequest but never got around to it.